index.bs

```diff
@@ -1817,11 +1884,12 @@ const c = builder.matmul(a, b);
 const graph = await builder.build({c});
 
 const bufferA = new Float32Array(sizeOfShape(descA.dimensions)).fill(0.5);
-const inputs = {'a': {data: bufferA}};
+const inputs = {'a': new MLTensor(bufferA)};
 // Pre-allocate output buffer for c.
-const outputs = {'c': {data: new Float32Array(sizeOfShape([3, 3]))}};
-await graph.compute(inputs, outputs);
-console.log(&#96;values: ${outputs.c.data}&#96;);
+const outputs = {'c': new MLTensor(new Float32Array(sizeOfShape([3, 3])))};
+graph.compute(inputs, outputs);
+const data = await outputs.c.data();
```

Comment:
I am not sure whether the `MLTensor` is a right solution for this pre-allocated outputs scenario. Will `outputs.c.data()` turn a newly created `ArrayBufferView` instead of downloading the result into the pre-allocated `ArrayBufferView`?

Reply:
As the outputs are given, I would suggest `compute` not to return the results. The calling `data()` of the given output `MLTensor` should download the results to the pre-allocated `ArrayBufferView` and return it (the same object). e.g.,
```js
const preallocatedBuffer = new Float32Array(sizeOfShape([3, 3]);
const outputs = {'c': new MLTensor(preallocatedBuffer))};
graph.compute(inputs, outputs); // compute does not return
const returnedBuffer = await outputs.c.data();
// returnedBuffer === preallocatedBuffer 
```